Orm/Xtensive.Orm.Tests.Core/Linq/LambdaExpressionFactoryTests.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2003-2010 Xtensive LLC.
+// All rights reserved.
+// For conditions of distribution and use, see license.
+// Created by: Denis Krjuchkov
```

Comment:
Copyright years are incorrect and probably just copy-pasted. Creation date is incorrect too. Please use yyyy.mm.dd format